docs/setup/using_in_browser.md

```diff
@@ -0,0 +1,10 @@
+[Initialization of library](./intialization.md) requires several properties of [signer](../api/classes/modules_signer.SignerService.md):
+
+- The signer meets [EIP-191: Signed Data Standard](https://eips.ethereum.org/EIPS/eip-191)
+- Public key. This is legacy parameter, which was used to initialized DID document
+
+These properties are not directly exposed by all signers, such as Metamask, and thus are determined in runtime by sending test sign request.
+In order to improve user expirience when library is used in browser it tries to get these parameters from local storage.
+Because library doesn't change storage application should set `isEthSigner` and `PublicKey` storage variables on login.
```

Comment:
What do you mean by "the library doesn't change storage" here? I guess you mean that application javascript can't set local storage? I don't think this is the case (https://developer.mozilla.org/en-US/docs/Web/API/Storage/setItem), so I'm a bit confused as to what this line means.
```suggestion
Because the library doesn't change storage, the application should set `isEthSigner` and `PublicKey` local storage variables on login.
```

Reply:
Yes, I meant local storage. By 
> the library doesn't change storage
I mean we prefer that the library doesn't modify the local storage and let the application manage the session as needed. I will paraphrase this